src/app/personside/infotabs/meldinger/traadliste/besvarflere/BesvarFlere.tsx

```diff
@@ -0,0 +1,165 @@
+    > *:last-child {
+        background-color: ${theme.color.navLysGra};
+        padding: ${theme.margin.layout};
+    }
+`;
+
+const TraadStyle = styled.div`
+    display: flex;
+    > *:first-child {
+        flex: 30% 1 1;
+    }
+    > *:last-child {
+        flex: 70% 1 1;
+    }
+    > * {
+        overflow: auto;
+    }
+`;
+
+const TraadlistStyle = styled.ol`
+    display: flex;
+    background-color: white;
+    flex-direction: column;
+    list-style: none;
+    > * {
+        border-bottom: ${theme.border.skilleSvak};
+    }
+`;
+
+const TraadVisningStyle = styled.section`
+    padding: ${theme.margin.layout};
+    flex-grow: 1;
+    > *:last-child {
+        margin-top: ${theme.margin.layout};
+    }
+`;
+
+const StyledCheckbox = styled(Checkbox)`
+    padding: 1rem;
+    transform: translateY(-0.5rem);
+`;
+
+const TittelWrapper = styled.div`
+    background-color: ${theme.color.navLysGra};
+    padding: 1.25rem ${theme.margin.layout};
+`;
+
+function hentTemagruppe(traader: Traad[]) {
+    return traader[0].meldinger[0].temagruppe.toString();
+}
+
+function hentSlaasammenOppgaver(traader: Traad[]): SlaaSammenOppgave[] {
+    return traader.reduce((acc: SlaaSammenOppgave[], traad: Traad) => {
+        return acc.concat(
+            traad.meldinger.map(melding => ({
+                oppgaveId: melding.oppgaveId!,
+                henvendelsesId: traad.traadId,
+                meldingsId: melding.id
+            }))
+        );
+    }, []);
+}
+
+function Meldingsvisning({ traad }: { traad: Traad }) {
+    const meldinger = traad.meldinger.map(melding => <EnkeltMelding melding={melding} sokeord={''} />);
+
+    return <TraadVisningStyle>{meldinger}</TraadVisningStyle>;
+}
+
+function BesvarFlere(props: Props) {
+    const dispatch = useDispatch();
+    const slaaSammenResource = useSelector((state: AppState) => state.restResources.slaaSammen);
+    const [valgteTraader, setValgteTraader] = useState<Traad[]>([]);
+    const [traadSomSkalVises, setTraadSomSkalVises] = useState<Traad>(props.traader[0]);
+
+    function onCheckTraad(traad: Traad) {
+        if (valgteTraader.includes(traad)) {
+            const nyListe = valgteTraader.filter(t => t.traadId !== traad.traadId);
+            setValgteTraader(nyListe);
+        } else {
+            const nyListe = [...valgteTraader, traad];
+            setValgteTraader(nyListe);
+        }
+    }
+
+    const traadkomponenter = props.traader.map(traad => {
+        const checkbox = (
+            <StyledCheckbox
+                label={''}
+                checked={valgteTraader.map(traad => traad.traadId).includes(traad.traadId)}
+                onChange={() => onCheckTraad(traad)}
+            />
+        );
+        return (
+            <TraadListeElement
+                traad={traad}
+                key={traad.traadId}
+                erValgt={traadSomSkalVises.traadId === traad.traadId}
+                onClick={() => setTraadSomSkalVises(traad)}
+                sokeord={''}
```

Comment:
du kan jo rename denne til `highlight` og gjøre den optional

Reply:
`sokeord` brukes nå i `TraadListeElement` for å beholde fokus på søkefeltet `props.sokeord.length > 0`. Så blir nok feil å rename til `highlight`

Reply:
aha. Men tror jeg likevel ville vurdert å skrive det om. Trådlisteelement bør egentlig ikke ha noe forhold til søk. Det Trådlisteelement trenger å vite er hvorvidt den skal ta fokus eller ikke. Nå bruker den søkeord til å beregne dette på et vis, men egentlig burde den bare få en boolean som sier `taFokus`, event `kanTaFokus`